source/index.ts

```diff
@@ -0,0 +1,5 @@
+import {Conf} from './conf';
+
+export {Conf};
+
+export default Conf;
```

Comment:
I don't really see the point of this file? Why can't the `conf.ts` contents be in here?

Reply:
This was some eslint thing, because we require Conf in `types.ts` as well. But it's a bit useless to have it in another file, so I'll place it back in index.ts